tekton/tasks/source-container-exit.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: source-container-exit-0-1  # dot is not allowed in the name
+spec:
+  description: >-
+    OSBS task for executing source container exit task
+  params:
+    - name: OSBS_IMAGE
+      description: The location of the OSBS builder image (FQDN pullspec)
+      type: string
+    - name: USER_PARAMS
+      type: string
+      description: User parameters in JSON format
+
+  workspaces:
+    - name: ws-build-dir
+    - name: ws-context-dir
+    - name: ws-registries-secret  # access with $(workspaces.ws-registries-secret.path)/token
+    - name: ws-koji-secret  # access with $(workspaces.ws-koji-secret.path)/token
+    - name: ws-reactor-config-map
+
+  steps:
+    - name: source-container-exit-image
+      image: $(params.OSBS_IMAGE)
+      resources:
+        requests:
+          memory: 3000Mi
+          cpu: 500m
+        limits:
+          memory: 8600Mi
+          cpu: 1300m
```

Comment:
This could probably use the same resources as the binary container exit task. Only the source-container-build task should need the extra resources